high-level-tests/e2e/cypress/integration/pix-app/a11y.test.js

```diff
@@ -23,11 +23,14 @@ describe('a11y', () => {
 
   describe('Not authenticated pages', () => {
     const notAuthenticatedPages = [
+      { url: '/campagnes/WALL/presentation' },
+      { url: '/changer-de-mot-passe' },
       { url: '/connexion' },
       { url: '/inscription' },
       { url: '/mot-de-passe-oublie' },
       { url: '/nonconnecte' },
-      { url: '/campagnes/WALL/presentation' },
+      { url: '/recuperer-mon-compte', skipFailures: true },
```

Comment:
**question**
Il fait quoi exactement le `skipFailures` ? Tu dis dans la description que ça remonte les erreurs ?
Ca vient de `axe` ou c'est du cypress ?

Reply:
https://github.com/component-driven/cypress-axe#skipfailures-optional-defaults-to-false

Ca vient de la libraire cypress-axe, qui remonte les erreurs.